repair/repair.cc

```diff
@@ -114,6 +114,21 @@ class node_ops_metrics {
         return removenode_total_ranges == 0 ? 1 : float(removenode_finished_ranges) / float(removenode_total_ranges);
     }
     float repair_finished_percentage();
+public:
+    void init() {
+        bootstrap_total_ranges = 0;
```

Comment:
All these variables are already initialized as 0, so all these assignments are redundant... 
E.g., we already had
```
    uint64_t bootstrap_total_ranges{0};
    uint64_t bootstrap_finished_ranges{0};
    uint64_t replace_total_ranges{0};
```

I don't like such duplication because it's a sure way to forget to update one of them in the future. 

Wouldn't a do-nothing init() work just the same - if all you need is for the variable to be initialized? If I'm right, you don't even need the new init function - you can call any existing function. Whether it's a do-nothing init or calling a silly existing function, I would add a comment explaining why.

Reply:
Yes, I checked that a do nothing function is enough. I would prefer calling a dummy function instead of a random existing function. I added some comments in v2.